src/components/tile.tsx

```diff
@@ -35,42 +40,73 @@ type Props = ButtonHTMLAttributes<HTMLButtonElement> & {
   width?: string;
 };
 
+const nextHeading = (level: HeadingLevels) => {
+  if (level === 'h6') {
+    throw new Error('no lower heading level than h6');
+  }
+  return `h${+level[1] + 1}`;
+};
+
 export const Tile: FC<Props> = ({
   title,
+  headingLevel = 'h3',
   subtitle,
-  description,
   backgroundColor,
   backgroundImage,
   gradient = false,
   width,
   className,
   style,
+  children,
   ...props
-}) => (
-  <button
-    type="button"
-    className={classNames(className, {
-      tile: 'tile',
-      'tile-gradient': gradient,
-    })}
-    style={
-      {
-        ...style,
-        '--tile-background': backgroundColor,
-        width: width || '100%',
-      } as CSSProperties
-    }
-    {...props}
-  >
-    <section className="tile__background-image">{backgroundImage}</section>
-    <section className="tile__content">
-      <h3 className="tile__header">{title}</h3>
-      {subtitle && <h5 className="tile__subtitle">{subtitle}</h5>}
-      {description && (
-        <small className="tile__description">{description}</small>
+}) => {
+  const isExternal = 'url' in props;
+
+  const linkTarget = (
```

Comment:
Wonder if "tileBody" or "tileMain" would be a better name?

Reply:
changed